operators/multiclusterobservability/pkg/certificates/certificates.go

```diff
@@ -274,6 +277,7 @@ func createCertSecret(c client.Client,
 			crtSecret.Data["ca.crt"] = caCertBytes
 			crtSecret.Data["tls.crt"] = certPEM.Bytes()
 			crtSecret.Data["tls.key"] = keyPEM.Bytes()
+			crtSecret.Labels["cluster.open-cluster-management.io/backup"] = ""
```

Comment:
we don't need this since the label already injected when the secret created

Reply:
Removed in latest commit